stores/moderationSubmissionsStore.ts

```diff
@@ -0,0 +1,55 @@
+import { gql } from "graphql-request";
+import { defineStore } from "pinia";
+import { Locale, Submission } from "~/typedefs/gqlTypes.js";
+import { gqlClient } from "../utils/graphql.js";
+import { Ref, ref } from "vue";
+
+export const useModerationSubmissionsStore = defineStore(
+    "submissionsStore",
+    () => {
+        const submissionsData: Ref<Submission[]> = ref([]);
```

Comment:
nit: omg there's semicolons on almost EVERY line. 😿 😢 😭 🥹

Reply:
I will do my best to fix this by tomorrow